### PR TITLE
Clean up formatting and add ticket refs for todos.

### PR DESCRIPTION
**Pull request by Camwyn**
Ticket refs in the todo comments.

Includes/requires: https://github.com/moderntribe/tribe-common/pull/1500 (merged)

Set against master until this gets an OK and a release.

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -1474,7 +1474,7 @@ public function get_event_label_singular() {
 		/**
 		 * Load the day view template tags
 		 * Loaded late due to potential upgrade conflict since moving them from pro
-		 * @TODO move this require to be with the rest of the template tag includes in 3.9
+		 * @todo move this require to be with the rest of the template tag includes in 3.9
```
Comment (tr1b0t) [inline src/Tribe/Main.php:1477]: :no_entry_sign: There must be exactly one blank line before the tags in a doc comment

<sub>hash: 73701ec576209680ac7b0f988ac7d0fb</sub>

File: src/Tribe/Query.php [hunk 2]
```diff
@@ -342,7 +342,7 @@ public static function pre_get_posts( $query ) {
 
 				$skip_event_display_filters = is_admin() && $query->is_main_query() && ! tribe_is_ajax_view_request();
 
-				//@todo stop calling EOD cutoff transformations all over the place
+				//@todo [BTRIA-607]: Stop calling EOD cutoff transformations all over the place.
```
Comment (tr1b0t) [inline src/Tribe/Query.php:345]: :no_entry_sign: No space found before comment text; expected "// @todo [BTRIA-607]: Stop calling EOD cutoff transformations all over the place." but found "//@todo [BTRIA-607]: Stop calling EOD cutoff transformations all over the place."

<sub>hash: 7c8895804607610926f27d857eb35730</sub>

File: src/Tribe/Views/V2/Repository/Event_Period.php [hunk 3]
```diff
@@ -344,7 +344,7 @@ public function not_in( $post_ids ) {
 	 * @return Tribe__Repository__Read_Interface
 	 */
 	public function parent( $post_id ) {
-		// TODO: Implement parent() method.
+		// @todo [BTRIA-595]: Implement parent() method.
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Repository/Event_Period.php:347]: :warning: This comment is 42% valid code; is this commented out code?

<sub>hash: 5b99b9b945bdea4dbd65df15ba459003</sub>

File: src/Tribe/Views/V2/Rest_Endpoint.php [hunk 4]
```diff
@@ -167,7 +167,7 @@ public function register() {
 		return register_rest_route( static::ROOT_NAMESPACE, '/html', [
 			// Support both GET and POST HTTP methods: we originally used GET.
 			'methods'             => [ Server::READABLE, Server::CREATABLE ],
-			 // @todo  Make sure we do proper handling of cache longer then 12h.
+			 // @todo [BTRIA-600]:  Make sure we do proper handling of caches longer then 12h.
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Rest_Endpoint.php:170]: * :warning: Found precision alignment of 1 spaces.
* :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 13

<sub>hash: 880854b8921dbaa68bc4fc9b41891288</sub>

File: src/functions/template-tags/date.php [hunk 5]
```diff
@@ -94,13 +94,13 @@ function tribe_event_is_on_date( $date = null, $event = null ) {
...
 		if ( ! empty( $event->_end_date_fixed ) ) {
-			// @todo remove this once we can have all day events without a start / end time
+			// @todo [BTRIA-613]: remove this once we can have all day events without a start / end time.
 			$event_end = date_create( date( Tribe__Date_Utils::DBDATETIMEFORMAT, $event_end ) );
 			$event_end->modify( '+1 day' );
 			$event_end    = $event_end->format( 'U' );
 		}
 
-		/* note:
+		/* Note:
```
Comment (tr1b0t) [inline src/functions/template-tags/date.php:103]: :no_entry_sign: Block comment text must start on a new line

<sub>hash: 46aba9deb48d6a5861216c60c1341780</sub>

File: src/functions/template-tags/ical.php [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-// @TODO two functions that do the exact same thing
+// @todo [BTRIA-614]: two functions that do the exact same thing
```
Comment (tr1b0t) [inline src/functions/template-tags/ical.php:3]: :no_entry_sign: You must use "/**" style comments for a file comment

<sub>hash: 444bb2ba49bbe71fe53848bb5ceef829</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 5 error(s)
:warning: 2 warning(s)

File: src/Tribe/Main.php (hunk 1 above)
Comment (tr1b0t) [inline src/Tribe/Main.php:1477]: :no_entry_sign: There must be exactly one blank line before the tags in a doc comment

<sub>hash: 73701ec576209680ac7b0f988ac7d0fb</sub>

File: src/Tribe/Views/V2/Repository/Event_Period.php (hunk 3 above)
Comment (tr1b0t) [inline src/Tribe/Views/V2/Repository/Event_Period.php:347]: :warning: This comment is 42% valid code; is this commented out code?

<sub>hash: 5b99b9b945bdea4dbd65df15ba459003</sub>

File: src/Tribe/Views/V2/Rest_Endpoint.php (hunk 4 above)
Comment (tr1b0t) [inline src/Tribe/Views/V2/Rest_Endpoint.php:170]: * :warning: Found precision alignment of 1 spaces.
* :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 13

<sub>hash: 880854b8921dbaa68bc4fc9b41891288</sub>

File: src/functions/template-tags/date.php (hunk 5 above)
Comment (tr1b0t) [inline src/functions/template-tags/date.php:103]: :no_entry_sign: Block comment text must start on a new line

<sub>hash: 46aba9deb48d6a5861216c60c1341780</sub>

File: src/functions/template-tags/ical.php (hunk 6 above)
Comment (tr1b0t) [inline src/functions/template-tags/ical.php:3]: :no_entry_sign: You must use "/**" style comments for a file comment

<sub>hash: 444bb2ba49bbe71fe53848bb5ceef829</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 4 error(s)
:warning: 2 warning(s)

**Review by lucatume — COMMENTED** [review]
This is great work, should this not be re-based against `release/B20.11`?

**Review by bordoni — APPROVED** [review]

**Pull request merged**